package/testing/Scripts/get-zeek-env

```diff
@@ -30,7 +30,7 @@ else
         # Combine the local tree and the system-wide path. This allows
         # us to test on a local build or an installation made via zkg,
         # which squirrels away the build. --cpk
-        echo "$(cd "${base}/../.." && pwd):$(zeek-config --plugin_dir)"
+        echo "$(cd "${base}/../../build" && pwd):$(zeek-config --plugin_dir)"
```

Comment:
This seems to have zero effect, why are we even adding this? Could you either add a test which depends on this, or alternatively just remove this code (ideally in a separate commit)?

Reply:
Confine the plugin/hlto search to the `/build` directory (think `cp build build-1` workflows causing the wrong plugin/hlto to be loaded). I can add a test that renames the build directory and would trigger test failures. Would that work?

https://github.com/zeek/btest/issues/98

But, it actually should be more like `$(cd "${base}/../../build" && pwd)/build`.